release_notes.md

```diff
@@ -1,21 +1,10 @@
 # What's New
```

Comment:
Improved validations like it will check for connector and its configuration is present or not.